lib/jsonapi/paginator.ex

```diff
@@ -0,0 +1,15 @@
+defmodule JSONAPI.Paginator do
+  @moduledoc """
+  Pagination strategy behaviour
+  """
+
+  @type links :: %{
+          first: String.t() | nil,
+          last: String.t() | nil,
+          next: String.t() | nil,
+          prev: String.t() | nil
+        }
+
+  @callback paginate(data :: term, view :: atom, conn :: Plug.Conn.t(), page :: JSONAPI.Page.t()) ::
+              links
```

Comment:
I don't think we want this on a newline, could you check that there's not extra whitespace or returns here?

Reply:
`mix fomat --check-formatted` fails if everything goes on the same line, not sure how to go about this one.

Reply:
Aliased the modules to shorten line length seems like a goog compromise